src/main/python/cfn_sphere/transform.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+import collections
+import yaml
+import os.path
+
+def merge_includes(data, path):
+    merged_dict = {}
+
+    includes = data.get('include', {})
+
+    for include in includes:
+        with open(f'{path}/{include}', 'r') as include_stream:
```

Comment:
@rayhwang-kcom Apparently line 13 breaks the build. "Invalid Syntax"

Reply:
It's because it's checking using python2 which doesn't know about f-strings

Reply:
We should refactor to be 2.7 compatible.